netlogo-gui/src/main/app/InterfaceTab.scala

```diff
@@ -31,6 +31,7 @@ class InterfaceTab(workspace: GUIWorkspace,
     // way so that only a vertical scrollbar is really needed - ST 7/13/04
     ScrollPaneConstants.VERTICAL_SCROLLBAR_ALWAYS,
     ScrollPaneConstants.HORIZONTAL_SCROLLBAR_AS_NEEDED)
+  scrollPane.setBorder(new javax.swing.border.EmptyBorder(0, 0, 0, 0))
```

Comment:
That's a good point. I'm finishing up some changes to get this looking right in linux, I'll have this use BorderFactory as well.